Libraries/Opc.Ua.Server/Configuration/TrustList.cs

```diff
@@ -538,12 +539,21 @@ private ServiceResult RemoveCertificate(
                             {
```

Comment:
the exception should be catched in enumerateCRLs, just don't return it if it is crap

Reply:
@mregen I would make the EnsureDecoded Function of the Crl class public in this case?

Reply:
why? Can't you catch the exception in enumerateCRL?

Reply:
EnumerateCrl will not try to decode the crls as decoding only happens when you access a property of the crl. Of course the same effect could by achieved trying to read out any of the crls properties but would be less readable code.

Reply:
good point, I think we have to bite the bullet and call EnsureDecoded in the constructor, otherwise code presumes it deals with a valid CRL, when it doesn't.

Reply:
agree